### PR TITLE
KEYCLOAK-13562 removing duplicate text

### PR DESCRIPTION
**Pull request by andymunro**
@abstractj Can you check this looks right?

### CODE REVIEW

**Comment by andymunro** [comment]
@mposolda Can you approve this?  I removed the duplicated content.

**Review by mposolda — DISMISSED** [review]
@andymunro Sure, Thanks!

File: openshift/topics/reference.adoc [hunk 1]
```diff
@@ -58,61 +58,6 @@ pulled from the repository manager, instead of the default public repositories.
...
-MAVEN_MIRROR_URL=http://10.0.0.1:8080/repository/internal/
-----
-. Schedule new build of the application
-
-NOTE: During application build, you will notice that Maven dependencies are
-pulled from the repository manager, instead of the default public repositories.
-Also, after the build is finished, you will see that the mirror is filled with
-all the dependencies that were retrieved and used during the build.
-
-[[env_vars]]
```
Comment (hmlnarik) [inline openshift/topics/reference.adoc:115]: This line must remain there

**Review by hmlnarik — CHANGES_REQUESTED** [review]
Looks good, yet one line should remain

**Comment by abstractj** [comment]
@andymunro could you please follow-up on the changes requested by @hmlnarik ?

File: openshift/topics/reference.adoc (hunk 1 above)
Comment (andymunro) [reply openshift/topics/reference.adoc:115]: @hmlnarik, @abstractj  I did add back these lines to the section.  Do they somehow need to be pushed?

=== Environment Variables

==== Information Environment Variables

Comment (abstractj) [reply openshift/topics/reference.adoc:115]: @andymunro if you added these lines back, yes, you need to push the changes again. //cc @hmlnarik

Comment (andymunro) [reply openshift/topics/reference.adoc:115]: Okay, I am not sure why this did not work before, but I added a space to the end of one line and pushed it again. Thanks, @abstractj.

Comment (hmlnarik) [reply openshift/topics/reference.adoc:115]: The `[[env_vars]]` line seems still be removed and needs to remain.

**Review by hmlnarik — CHANGES_REQUESTED** [review]
The `[[env_vars]]` line seems still be removed and needs to remain.

**Comment by abstractj** [comment]
@andymunro please check the comments that Hynek made.

**Comment by andymunro** [comment]
Thanks, @abstractj. Adding back the missing line.

**Comment by abstractj** [comment]
Changes look good to me know, but I will wait for @hmlnarik's input.

**Review by hmlnarik — APPROVED** [review]

**Review by abstractj — APPROVED** [review]

**Pull request merged**